src/server.ts

```diff
@@ -10,17 +11,21 @@ const PORT = 8080;
 const API_ROOT = '/api';
 const ENV = process.env.NODE_ENV || 'development';
 
-const app = express();
+createConnection().then(() => {
```

Comment:
Not _necessary_, but worth considering making a top-level async main function that awaits this call, and then making an await call to main at the top level to avoid this promise chain.

```ts
async function main() {
  await createConnection();
 
 // Blablabla
}

await main();
```